packages/salesforcedx-vscode-lwc/src/constants.ts

```diff
@@ -7,3 +7,7 @@
 
 export const ESLINT_NODEPATH_CONFIG = 'eslint.nodePath';
 export const LWC_EXTENSION_NAME = 'salesforcedx-vscode-lwc';
+
+export const log = (message: string) => {
```

Comment:
the test was stubbing the console.log method and it never seemed to work correctly.  I moved it out to this wrapper method so the log call could be stubbed.